README.md

```diff
@@ -25,7 +25,7 @@ python3 person_blocker.py -i images/img3.jpg -c '(128, 128, 128)' -o 'bus' 'truc
 ```
 
 * `-i/--image`: specifies the image file.
-* `-m/--model`: path to the pretrained COCO model weights (default: current directory): if not specified, it will download them automatically to the current directory if not already present (note: the weights are 258 MB!)
+* `-m/--model`: path to the pretrained COCO model weights (default: current directory): if not specified, the software will download the paths automatically to the current directory if not present (note: the weights are 258 MB!)
```

Comment:
> if not specified, the software will download the paths automatically to the current directory if not present

This is incorrect – the script doesn't download paths, it downloads a file containing the pretrained COCO model weights.

How about: "If a path is not specified and the file doesn't exist in the current directory, it will be downloaded automatically (note: download size is ~258 MB)"